db/people.yml

```diff
@@ -3,11 +3,12 @@ sharouna-mayer:
   aka: []
   avatar: https://avatars3.githubusercontent.com/u/58988624?s=400&u=6871fccc040af8766bebdb13c8629407d607b18b&v=4
   bio: Sani grew up in Kassel, Germany. He earned his Bachelors degree in nanoscience
-  in 2018. In his Bachelor thesis he worked on PDF analysis of electrochrmic thin films
-  in the group of Dorota Koziej. Ever scince he continued working in Dorotas group
-  as a studental research assistant, where he focuses on other electrochromic systems
-  and insitu imaging of nanostructures. He currently works in the group of Simon Billinge
-  for an external Masters procect.
+    in 2018 at the University of Hamburg. In his Bachelor thesis he worked on PDF analysis
+    of electrochrmic thin films in the group of Dorota Koziej. Ever scince he continued
+    working in Dorotas group as a studental research assistant, where he focuses on
```

Comment:
since this is public facing, maybe better to put ``Dr. Koziej's group``  Also just ``research assistant`` may be better, unless "studental research assistant" is an actual title for the position.